src/components/CareerDevelopment/CareerDevelopment.scss

```
@@ -0,0 +1,26 @@
+.career-development {
+  width: 640px;
+  display: flex;
+  // text-align: end;
+
+  &__add {
+    border: none;
+    border-radius: 6px;
+    background: var(--primary-light, #ebfffc);
+    color: var(--primary-dark, #00947e);
+    /* text bold */
+    font-family: Open Sans;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 150%; /* 24px */
```

```suggestion
    line-height: 24px;
```
размер шрифта-то тоже указан и тоже в пикселях.